CONTRIBUTING.md

```diff
@@ -0,0 +1,39 @@
+# Contributing to Bearded.Utilities
+
+You want to help us improve our little library? ~~That's awesome!~~ You're awesome!
+
+Whether you have a feature request, found a bug, want to add some code of your own or even just have a question - all of these are very welcome.
+
+Our library is pretty small, so let's try and go easy on the rules. :)
+
+
+## The basics
+
+- **Have a question?** [Open an issue!](https://github.com/beardgame/utilities/issues/new/choose)
+- **Found a bug?** [Open an issue!](https://github.com/beardgame/utilities/issues/new/choose)
+- **Would you like to request a feature, change or improvement?** [Open an issue!](https://github.com/beardgame/utilities/issues/new/choose)
+
+You could also always check out list of issues and see if there's any discussions you want to share your thoughts on.
+
+
+## You'd actually like to make changes yourself?
+
+Wow, that's even better!
+
+In principle this as easy as forking the repository, making some changes and [opening a pull request](https://github.com/beardgame/utilities/compare). From there, we'll review your PR as soon as we can manage, ask questions or request changes if necessary, and once we're happy merge it into our main branch. It's that easy!
+
+If you want to know ahead of time if we'd welcome a certain feature or change, you could always create an issue where you propose that change and mention that you'd like to implement it yourself. We'll let you know what we think as soon as we can. The main reason we might decline code is if we feel it doesn't fit with the purpose of our libraries, but we're very open to suggestions and good arguments.
+
+When writing code for Bearded.Utilities, keep the following things in mind:
+
+- We like pull requests to be as small as possible, but as large as necessary, to make them as easy to review as possible
+- We like organised, consistent, clean, readable and maintainable code, see what's already there for examples (or submit a change if you think you can improve it!)
+- We like modern C# features, type-safety and care about performance
+- We like tests
+
```

Comment:
What is somewhat missing right now is how to jump on issues that already exist, if a potential contributor wants to contribute, but doesn’t have a specific feature or bug in mind (e.g. because they want a shiny hacktoberfest t-shirt).

Many repos work with something like a “good first issue” label to promote that, though it may be overkill for this library.

Not a blocker to add a label, but at the very least worth mentioning that any unassigned issue is fair game, though they should probably always assign themselves if they start.

Reply:
I added the label, will mention it here and also mention issue assignments.